model_zoo/vision/alexnet.py

```diff
@@ -36,6 +37,31 @@
     default=data_reader_prototext, type=str,
     help='data reader prototext file (default: ' + data_reader_prototext + ')',
     metavar='FILE')
+parser.add_argument(
```

Comment:
I really like where this is going, but these parser arguments need to be moved to a common class that is shared by all models, but can also be extended in a model specific way.  Otherwise we will get the same type of duplication that we seen between this and the mnist example.